### DISCUSSION TITLE
Required Resources Definition for Edge Application Management

### DISCUSSION THREAD

**Discussion by z-reda** [camaraproject/EdgeCloud#297] [Ideas]
I would suggest that we follow the cloud native pattern of defining resouces.

The CPU power has to be defined in units like milliCPU (mCPU), which allows for finer granularity (e.g., 500m for half a CPU). This precision helps ensure that applications receive exactly the amount of CPU they need, preventing both over-provisioning and under-provisioning.

Same for memory it should be in megabytes (MB) rather than gigabytes (GB) to offer better granularity and more accurate control over memory allocation. 

This allows for more precise tuning of application performance and resource usage. #

**Comment by JoseMConde** [comment]
Hi @z-reda, thanks for your comment.
We are actually working on this topic. There is a [PR-280](https://github.com/camaraproject/EdgeCloud/pull/280) where Felipe and Luis are improving this parameter. The PR is not merged yet, so the code is not in the main branch. You can see the modified file inside the PR.
As you can see, we are implementing this kind of things.
![image](https://github.com/user-attachments/assets/e8caf5b6-1673-45ea-8dfc-0f31c9df36e6)
 Please take a look and feel free to comment or add anything you consider on this PR.

**Reply by rzaza** [reply]
I see!

Sure, will take a look at this PR and contribute. 

Thanks a lot @JoseMConde
